styleguide-app/Examples/Button.elm

```diff
@@ -0,0 +1,223 @@
+module Examples.Button exposing (Msg, State, example, init, update)
+
+{- \
+   @docs Msg, State, example, init, update,
+-}
+
+import Debug.Control as Control exposing (Control)
+import Html exposing (..)
+import ModuleExample as ModuleExample exposing (Category(..), ModuleExample, ModuleMessages)
+import Nri.Ui.AssetPath exposing (Asset)
+import Nri.Ui.Button.V2 as Button
+import Nri.Ui.Icon.V2 as Icon
+
+
+{-| -}
+type Msg
+    = SetState State
+
+
+{-| -}
+type State
+    = State (Control Model)
+
+
+{-| -}
+type ButtonType
+    = Button
+    | Link
+    | CopyToClipboard
+
+
+{-| -}
+example :
+    { r | teach_assignments_copyWhite_svg : Asset }
```

Comment:
❓ Not sure whether I'm missing a step in the build process, but the icons do not appear in the style guide for me

Reply:
Yeah, we don't have any icons in the noredink-ui repo at the moment. We should fix that, I'll create a separate PR for that.